app/services/alerts_service.rb

```diff
@@ -0,0 +1,18 @@
+class AlertsService
+  include UiServiceMixin
+
+  def initialize(controller)
+    @controller = controller
+  end
+
+  def all_data
+    {
+      :alerts => alerts
+    }.compact
+  end
+
+  def alerts
+    result = {}
+    result.values
```

Comment:
Won't this always return []?
[Sorry to barge in without reading most of this PR, ignore me if silly question]